CONTRIBUTING.md

```diff
@@ -5,8 +5,9 @@ If you have any questions, please reach out on the [HoloLens forums](https://for
 
 # Process
 
-1. [Make a proposal](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) (either new, or for one of the elements in our backlog)
-2. Implement the proposal and its tests.
+1. [Make a proposal](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues)
+> If you are implementing something from our backlog, you do not need to file a new proposal.
```

Comment:
maybe a bullet point instead?  It doesn't seem to fit right with the numbers, imo

Reply:
sure